#![allow(unused, unused_imports)]
use std::error::Error;


fn main() -> Result<(), Box<dyn Error>> {
    Ok(())
}